src/helpers/useResource.ts

```diff
@@ -26,10 +26,12 @@ const fetchResource = async (
         await OnewelcomeSdk.authenticateDeviceForResource(scopes);
       }
     }
-
-    const data = await OnewelcomeSdk.resourceRequest(type, resourceDetails);
-
-    setData(data);
+    const baseUrl = await OnewelcomeSdk.getResourceBaseUrl();
+    const data = await OnewelcomeSdk.resourceRequest(type, {
+      ...resourceDetails,
+      path: baseUrl + resourceDetails.path,
```

Comment:
a user should also be allowed to give an absolute path for fetching resources besides the resource base URL path; It is possible that they have multiple certificates which should allow them to also fetch resources beside the resource base URL

Reply:
This is part of the example app, so that's possible, this is a helper function to do resource requests

Reply:
then its oke 👍